### ISSUE TITLE
希望能加入应用分流

### ISSUE THREAD

**Issue by delphinuz** [Trojan-Plus-Group/trojan-plus-app#2]
目前好像没这功能。

另外也可加入连线测试，确定是否连线成功，还有服务器延长显示。

**Comment by yuchting** [comment]
Yes, these two features are much necessary for good proxy application, but the only limitation of implement is my spare time.
Hope you can understand. Of course they will come out if I have spare time.
